docs/adrs/accessing-dmr-and-classifier-from-centops.md

```diff
@@ -0,0 +1,39 @@
+# Architectural Decision Record For Non-Chatbot Participants
+
+## Issue
+
+Non-chatbot Participants (Classifier and DMR) can have multiple instances in CentOps.  When dependent participants call CentOps to determine the endpoints for these special participants if several Dmr or Classifiers are returned by the CentOps Api.
+
+* CentOps doesn't currently have a way to filter Participants by type.
+* CentOps doesn't currently have a way to return the 'best' Dmr or Classifier based on properties of the caller.
+
+## Decision
+
+Implement an API which allows for filtering by Participant type:
+
+```url
+/public/participants?type=Dmr
+```
+
+The caller simply selects the a DMR or Classifier from the list of participants of those types.  To avoid sending unused data on the wire an additional filter 'count' could be implemented.

```

Comment:
```suggestion
The caller simply selects a DMR or Classifier from the list of participants of those types.  To avoid sending unused data on the wire an additional filter 'count' could be implemented.
```